src/pdc_osti/poster.py

```diff
@@ -173,6 +173,11 @@ def no_empty_cells(series):
     def _fake_post(self, records):
         """A fake JSON response that mirrors OSTI's"""
         self.log.info("[bold yellow]Fake posting")
+        try:
+            ostiapi.datatoxml(records)  # Check that JSON can be parsed into XML
+        except AttributeError:
+            raise AttributeError("Failure to load data into XML")
```

Comment:
I'm not sure why the `try-except` wrapper is necessary... Is there information in the original `AttributeError` that needs to be hidden?

Reply:
@mccalluc, thanks for your question. One of the breaking changes with `py3.10` is with the `collections` library. Attempting to run `dicttoxml <= 1.7.4` would break. Within the CI, we do a "dry-run" execution. As it turns out, this doesn't trigger parsing the JSON data to XML, which is something we should check as it is done with real data submission. To be fair, the unit testing should happen within `ostiapi` but that has been mostly unmaintained.

Thus, I added it here to ensure coverage completeness.